HomeTask50/HomeTask50_sub9/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace HomeTask50_sub9
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            IWebDriver driver;
+            ChromeOptions options = new ChromeOptions();
+            options.AddArgument("--incognito");
+            driver = new ChromeDriver(options);
+            driver.Url = "https://demo.seleniumeasy.com/table-sort-search-demo.html";
+            EmployeesTable emp = new EmployeesTable(driver,50.0);
+          //  emp.GetPagination();
+            var test = emp.AllEmployees();
+            var test1 = emp.GetSpecificEmployees(20, 500000);
+            Console.WriteLine("");
```

Comment:
All your tests SHOULD BE wrapped into method with [Test] annotation! Don't you Main method as a way to create test methods.